docs/QuestionsIssuesPrs.md

```diff
@@ -88,7 +88,9 @@ additional details provided by the specific project.
 1. After creating the PR, 
    1. Wait for CI (i.e. Travis, AppVeyor) to run tests/checks against your PR and report status. 
       If any errors are reported, fix those problems and push the fixes to the same branch.
-   1. When there are no more CI errors, post a comment to inform the dev team that the PR is ready for review. 
+   1. If the PR is big (e.g. affecting more than 100 lines), refactor the commits in the 
+      [multi-step PR format](HowToGuides.md#create-a-multi-step-pr).
+   1. When there are no more CI errors, post a comment to inform the dev team that the PR is ready for review.
```

Comment:
If we are using the `CanIHasReview` tool, then there is no need to inform the dev team that the PR is ready for review. (Since the `s.ToReview` label will indicate that).

So, we may want to add a `If the multi-step PR format is not used, and there are no more CI errors, then post a comment...`